api/fixtures/example.go

```diff
@@ -154,6 +156,10 @@ aws_secret_access_key = %s
 		sshKeyReference = corev1.LocalObjectReference{Name: sshKeySecret.Name}
 	}
 
+	var etcdStorgageClass *string = nil
+	if len(o.EtcdStorageClass) > 0 {
```

Comment:
Nit: Comparing strings against an empty string rather than using `len` makes it immediately obvious to the reader of your code that what you are checking is a string and not one of string, array or map